Torch.Server/Views/ModListControl.xaml

```diff
@@ -0,0 +1,126 @@
+﻿<UserControl x:Class="Torch.Server.Views.ModListControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:viewModels="clr-namespace:Torch.Server.ViewModels"   
+             xmlns:s="clr-namespace:System"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+             MouseMove="UserControl_MouseMove">
+    <!--<UserControl.DataContext>
+        <viewModels:ConfigDedicatedViewModel />
+    </UserControl.DataContext>-->
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="Resources.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+            <Style TargetType="Grid" x:Key="RootGridStyle">
+                <Style.Triggers>
+                    <DataTrigger Binding="{Binding Mode=OneWay, UpdateSourceTrigger=PropertyChanged, BindingGroupName=RootEnabledBinding}" Value="{x:Null}">
+                        <Setter Property="IsEnabled" Value="False"/>
+                    </DataTrigger>
+                </Style.Triggers>
+            </Style>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <UserControl.DataContext>
+        <viewModels:ConfigDedicatedViewModel />
+    </UserControl.DataContext>
+    <Grid Style="{StaticResource RootGridStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="500px"/>
```

Comment:
Be careful with hardcoded pixel widths. Make sure your control stays legible when the window is resized.

Reply:
Well, if it's resized to be _smaller_ than 500px, the whole datagrid won't be visible, but I didn't consider this a big problem.
Other than that the right pane (contianing steam workshop description is using up the rest of the width. I set a specific width mainly, because otherwise the url text box (below the datagrid) would adapt its width to it's content which looks awful - and I didn't find a better way to prevent that.

I could consider using `MaxWidth` instead though... What do you think?

Reply:
Okay, before I go and change it, could you please have a quick look at Steamkit2's web API example and confirm that I should not use their recommended way of doing it?
The example I'm referring to is [this one](https://github.com/SteamRE/SteamKit/blob/master/Samples/6.WebAPI/Program.cs).

Unfortunately, these examples are the only source of documentation on Steamkit2 altogether (except the XML documentation comments and the source), so doing anything different is always a pain in the ass.

Reply:
I wrapped it in a ScrollViewer now. MaxWidth doesn't help either, because the data grid is best viewed in a fixed width (or only a small range, e.g. 300px-500px), while the description area benefits from growing. The scroll bar will now be visible when the window is resized to a width below 700px.